src/providers/stripe_sca.js

```diff
@@ -0,0 +1,222 @@
+const libraryPaths = [
+  'https://js.stripe.com/v3/'
+];
+const DOMAIN = "https://app.thebookingfactory.com";
```

Comment:
Remember about WhiteLabel solutions.
Don't hardcode TheBookingFactory domain name.

Reply:
add same approach for domain configuration as in public-booking-widget